spec/features/admin/configuration/dynamic_content_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
```

Comment:
please try to run rubocop on your file with either in your ide or by running
bundle exec rubocop <file_path>

it will tell you about extra empty lines. there should be only one line here.

Reply:
Ran Rubocup - thanks! - and solved all offenses but one:

`Style/FrozenStringLiteralComment: Missing frozen string literal comment.
`

which seems to require adding this in the first line:
`# frozen_string_literal: true`

I found this as well on the file `spec/features/consumer/shopping/checkout_stripe_spec.rb`
I'm guessing this is not a big issue.